### PR TITLE
Filter duplicate chunks, correctly detect finished builds, and stream logs fast at the end

### PR DESCRIPTION
**Pull request by mdlavin**

### CODE REVIEW

File: src/build.js [hunk 1]
```diff
@@ -70,8 +83,21 @@ const followBuild = async ({ appSlug, buildSlug, client }, options = {}) => {
...
     // If the build is completed and there are no more log chunks
-    if (!!attributes.build_finished && response.data.log_chunks.length === 0) {
-      // No need to keep watching a completed build
+    if (attributes.finished_at) {
+      if (response.data.log_chunks.length === 0) {
+        // No need to keep watching a completed build
+        break;
+      }
+
+      // If the build is done and chuncks are being seeing, then pull them
```
Comment (mdlavin) [inline src/build.js:92]: I see the "chuncks" but I already pushed a version tag =(

**Review by aecorredor — APPROVED** [review]

File: src/build.js [hunk 2]
```diff
@@ -70,8 +83,21 @@ const followBuild = async ({ appSlug, buildSlug, client }, options = {}) => {
...
+      // as fast as possible
+      interval = 0;
+    }
+
+    // If the log was being followed and it switched to streaming before the
```
Comment (mdlavin) [inline src/build.js:97]: Another typo. "streaming" should be "archived" I'll follow up with a fix for both

File: src/build.js (hunk 1 above)
Comment (aecorredor) [reply src/build.js:92]: another tyny one: `being seeing`. I'm guessing it should be `being seen`

Comment (mdlavin) [reply src/build.js:92]: Will fix that too

**Pull request merged**